src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java

```diff
@@ -40,15 +40,11 @@ public void testAddUsers() {
         user2Majors.add("Religion");
         // Add 2 initially
         ctl.addUser(
-            ds, /*userID=*/ 1l, /*AuthID=*/ 2l,
-            /*userName=*/ "Sarah", /*userYear=*/ 2022,
-            /*userMajors=*/ user1Majors, /*skills=*/ Skills.LEADERSHIP.name(),
-            /*userTotalCompTasks=*/ 3);
+            ds, new UserData(1l, 2l, "Sarah", 2022,
```

Comment:
testAddUsers(com.rtb.projectmanagementtool.user.UserControllerTest): java.lang.Long cannot be cast to java.lang.Integer

I can't quite figure out which param I passed in this is coming from ...